pnpm-lock.yaml

```diff
@@ -1,1486 +1,77 @@
-lockfileVersion: '9.0'
```

Comment:
Please checkout this file and reinstall dependencies. I'm using an older pnpm version than you. The diff doesn't have to be this large.